verify.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Should we add `set -euo pipefail` here, so the overall script fails if any of the `nvm` or `npm` commands fail, or if the environment is somehow not set up correctly?